test/compression.js

```diff
@@ -676,6 +676,35 @@ describe('compression()', function () {
       .end()
     })
   })
+
+  describe('when callbacks are used', function () {
+    it('should call the passed callbacks', function(done){
+      var callbacks = 0;
+      var server = createServer({ threshold: '1kb' }, function (req, res) {
+        res.setHeader('Content-Type', 'text/plain')
+        res.write('Hello', null, function(){
+          callbacks++
+          res.flush()
```

Comment:
Good point, I'll remove it. I was simply including `.flush()` for completeness since nothing makes it to the client without the call.